static/app/utils/analytics/workflowAnalyticsEvents.tsx

```diff
@@ -52,6 +52,7 @@ export type TeamInsightsEventParameters = {
     session_id: string;
     wizard_v3: string;
   };
+  'project.create_page_viewed': {};
```

Comment:
@Neo-Zhixing Maybe we should call this `project_creation_page.viewed` so we can group other events on that page with the same prefix.